tkcalendar.py

```diff
@@ -104,6 +105,14 @@ def __init__(self, master=None, **kw):
         # --- date
         today = self.date.today()
 
+        try:
+            self._textvariable = kw.pop("textvariable")
+#            if ( self._datevariable is not None ):
+#              if not ( isinstance( self._datevariable, tkinter.Variable     ) ):
+#                raise TypeError( "tkinter.Variable type expected, {} given.".format( type( self._datevariable ) ) )
+        except KeyError:
+            self._textvariable = None
+
```

Comment:
No need to use a try/except block here, instead use `pop(key, fallback)` to avoid raising the `KeyError`. In addition, I think it would be simpler to create a `StringVar` to avoid testing its existence before changing its content later in the code.

In addition, right now, the textvariable does not work like the one for an Entry, because it is only modified by a new selection in the calendar, but changing the variable content does not change the calendar selection. So I will add a trace to do it.